materialize/resources/resource_connection_postgres.go

```diff
@@ -123,6 +130,7 @@ type ConnectionPostgresBuilder struct {
 	postgresHost           string
 	postgresPort           int
 	postgresUser           string
+	postgresUserSecret     string
```

Comment:
Doesn't Postgres have some other option secret fields:
* SSL CERTIFICATE AUTHORITY

It would also probably be good to add some new tests for postgres